package.json

```diff
@@ -32,6 +32,7 @@
   ],
   "activationEvents": [
     "workspaceContains:jest.config.js",
+    "workspaceContains:jest.config.mjs",
     "workspaceContains:jest.json",
     "workspaceContains:node_modules/.bin/jest",
     "workspaceContains:node_modules/react-scripts/node_modules/.bin/jest",
```

Comment:
PS: I think vscode does not watch these folders. Perhaps because there is a blacklist, or something.